mlx/ops.h

```diff
@@ -828,6 +828,9 @@ array arccos(const array& a, StreamOrDevice s = {});
 /** Arc Tangent of the elements of an array */
 array arctan(const array& a, StreamOrDevice s = {});
 
+/** Arc Tangent of elements of two arrays */
```

Comment:
Clarify this docstring please: `Inverse tangent of the ratio of two arrays`. Also please use `a` and `b` here to follow the Python API.